beetmoverscript/script.py

```diff
@@ -335,30 +346,24 @@ def get_destination_for_private_repack_path(context, manifest, full_path, locale
 
     # pretty name the `target` part to the actual filename
     pretty_full_path = os.path.join(
-        os.path.dirname(full_path),
+        locale,
```

Comment:
Yeah, I agree, stashing this within `locale` seems like a hack for whoever reads the code for the first time but I think it's good enough for now to solve the problem. I'm curious how this is handled in the generate_beetmover-manifest section where the `locale` is essential but I'll glance in a bit at that particular part.